scripts/ingests/utils.py

```diff
@@ -56,6 +88,9 @@ def sort_sources(db, ingest_names, ingest_ras, ingest_decs, verbose=False, searc
             nearby_matches = db.query_region(location, radius=radius)
             if len(nearby_matches) == 1:
                 namematches = nearby_matches
+                coord_match = namematches[0]['source']
+                # Populate astropy table with ingest name and database name match
+                alt_names_table.append(Alt_names(coord_match, name))
             if len(nearby_matches) > 1:
                 print(nearby_matches)
                 raise Exception("too many nearby sources!")
```

Comment:
It's always better to raise a more specific exception (eg, TypeError, ValueError, RuntimeError) rather than the general Exception, but I don't know which one would be best.